lib/pure/times.nim

```diff
@@ -1121,12 +1300,119 @@ proc evaluateStaticInterval(interval: TimeInterval): Duration =
     minutes = interval.minutes,
     hours = interval.hours)
 
+proc between*(startDt, endDt:DateTime): TimeInterval =
+  ## Evaluate difference between two dates in ``TimeInterval`` format, so, it
+  ## will be relative.
+  ##
+  ## **Warning:** It's not recommended to use ``between`` for ``DateTime's`` in 
+  ## different ``TimeZone's``. If used in such manner with ``strict`` parameter 
+  ## set to ``true`` it will result in error.
+  ##
+  ## Set ``strict = false`` only if you know what you're doing! The results in
+  ## this case mey differ from that what you can expect!
```

Comment:
You forgot to update the comment, it should just warn that `a + between(a, b) == b` is only guaranteed when `a` and `b` are in UTC.

Reply:
Thanks!

Reply:
fixed